day_10/task_1/pipe_maze.cpp

```diff
@@ -1,16 +1,16 @@
 #include "pipe_maze.hpp"
 
-std::pair<int, int> find_starting_point(const PipeMaze& pipe_maze)
+MazePoint find_starting_point(const PipeMaze& pipe_maze)
 {
     for(auto i = 0U; i < pipe_maze.size(); i++)
     {
         for(auto j = 0U; j < pipe_maze[i].size(); j++)
         {
             if(pipe_maze[i][j] == 'S')
             {
-                return std::make_pair(i, j);
+                return MazePoint(i, j);
             }
         }
     }
-    return std::make_pair(-1, -1);
+    return MazePoint(-1, -1);
 }
```

Comment:
Missing newline

Reply:
done

Reply:
Is it really done? It seems to still miss. Could you add it, please? :)

Reply:
i missed that, sorry